src/scripts/main.js

```diff
@@ -1,3 +1,20 @@
 'use strict';
 
-// write code here
+const elementUl = [...document.querySelectorAll('li')];
+
+for (const elem of elementUl) {
+  const span = document.createElement('span');
+
+  elem.prepend(span);
+  span.append(span.nextSibling);
+
+  span.addEventListener('click', click => {
+    const item = click.target.closest('span');
+
+    if (item) {
+      const title = click.target.parentElement.children[1];
```

Comment:
```suggestion
      const title = event.target.parentElement.children[1];
```
переименуй в event, это общепринятая практика называть так этот объект

Reply:
у мене тоді лінтер свариться ((і не пускає 
![image](https://user-images.githubusercontent.com/51997778/132831225-68e7c101-2783-4c24-aab1-ae99d58dc3dc.png)